experimental/ast/type.go

```diff
@@ -12,6 +12,7 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
+//nolint:dupword // Disable for whole file, because the error is in a comment.
```

Comment:
Could use this comment elsewhere. That is definitely disappoint that there's no other way to suppress the warning in a doc comment 😞